SECURITY.md

```diff
@@ -2,7 +2,15 @@
 
 ## Supported Versions
 
-The .NET Core and ASP.NET Core support policy, including supported versions can be found at the [.NET Core Support Policy Page](https://dotnet.microsoft.com/platform/support/policy/dotnet-core).
+We only provide support for the latest version of `System.Device.Gpio`. Issues and contributions related to `Iot.Device.Bindings` are community-driven and not officially supported by the maintainers.
+
+## Considerations
+
+- Default Trust: By default, many IoT devices assume inputs are from trusted sources. This is a risky assumption because attackers can inject malicious data or manipulate signals.
+- Security Vulnerabilities: Attackers may craft inputs to exploit buffer overflows, force unexpected states, or gain unauthorized access to device functions.
+- Mitigation:
+  - Input Validation: Perform rigorous checks (e.g., sanitization, length checks, type validation) on every input—whether it’s sensor data, network traffic, or user-supplied information.
+  - Least Privilege: Run IoT components with minimal permissions. Ensure that any compromised part of the system cannot escalate privileges or affect other parts.

```

Comment:
I'm checking if we need to update below sections as well. I wanted to update above as well because currently it's not super clear if it's directed to contributors or consumers

Reply:
Security.md isn't intended to include content about how to write secure code. It is _solely_ intended for describing how to report security vulnerabilities. Increasing the scope of the file is not a good idea and takes away from this important topic.

It _is_ a good idea to explain how to write secure code. It should be in a docs directory, not in the root of the repo. It is not a topic that rises to needing to be in the root of a repo.